clusterloader2/pkg/measurement/common/profile.go

```diff
@@ -61,8 +61,9 @@ func (p *profileMeasurement) populateProfileConfig(config *measurement.Measureme
 	if p.config.provider, err = util.GetStringOrDefault(config.Params, "provider", config.ClusterFramework.GetClusterConfig().Provider); err != nil {
 		return err
 	}
-	if p.config.host, err = util.GetStringOrDefault(config.Params, "host", config.ClusterFramework.GetClusterConfig().GetMasterIp()); err != nil {
-		return err
+	p.config.hosts = config.ClusterFramework.GetClusterConfig().MasterIPs
+	if len(p.config.hosts) < 1 {
+		klog.Warningf("Profile measurements will be disabled due to no MasterIps: %v", p.config.hosts)
```

Comment:
NIT: If `len(p.config.hosts) < 1` then p.config.hosts = [] and the message will be always the same.
`Profile measurements will be disabled due to no MasterIps: []`

Maybe get rid of the %v part ?

Reply:
Thank you for the catch!